autonomy/cli/service.py

```diff
@@ -72,36 +75,41 @@ def activate(
 @click.option(
     "-i",
     "--instance",
+    "instances",
     type=str,
     required=True,
+    multiple=True,
```

Comment:
This was the reason why all 4 agent instances were not getting registered at the same time, although we were providing 4 agent instances the command was only taking one of them as input, this was my mistake on the design part